admin/autoupdater/src/uclient.c

```diff
@@ -175,6 +175,10 @@ int get_url(const char *url, void (*read_cb)(struct uclient *cl), void *cb_data,
 		goto err;
 	if (uclient_http_set_header(cl, "User-Agent", user_agent))
 		goto err;
+	if (firmware_version != NULL) {
+	    if (uclient_http_set_header(cl, "X-Firmware-Version", firmware_version))
+		    goto err;
```

Comment:
The indentation is off here, there should be one tab per level and no spaces.

Reply:
ok, fixed.